zipkin-web/src/main/resources/templates/v2/index.mustache

```diff
@@ -2,19 +2,11 @@
   <form class="form-inline" role="form">
     <div class="form-group">
       <select data-placeholder="Service Name" class="form-control input-sm" style="width: 234px;" name="serviceName" id="serviceName">
-        <option value=""></option>
```

Comment:
What is the expensive part of it? The call between the query server and the storage engine/cassandra or the browser rendering? It sounds strange that it takes such a long time... is it due to very many service names?